### ISSUE TITLE
April 2018 Compounds for hERG Evaulation

### ISSUE THREAD

**Issue by edwintse** [OpenSourceMalaria/Series4#37]
The team over at the Victor Chang Cardiac Research Institute here in Sydney recently acquired a new SyncroPatch high throughput screening robot for hERG evaluation late last year. We have decided on 9 compounds to run though their assay, shown below (hERG data on past compounds can be found in the Wiki [here](https://github.com/OpenSourceMalaria/Series4/wiki/hERG-Activity)):

![herg 2018](https://user-images.githubusercontent.com/18062981/38914205-d1fc6fdc-4321-11e8-85ab-ada3572af1f5.jpeg)

**700** and **784** were chosen as unique pheny ring derivatives with **698** as the comparison ([Labtrove](https://malaria.ourexperiment.org/biological_data/15741/Biological_Data_from_Dundee__November_2016_Compounds.html) and #8)
**XXX** is a recently synthesised DCNYS compound (#30)
**336** is the hERG evador (fingers crossed!) and **155** is the comparison (#22)
**341** is the recent highly potent compound made by @maratsydney (#26)
**100** is the lead Series 3 compound made by @mbhebhe
**14-1** is a lead compound from the Open Source Mycetoma project made by last years honours student Hung ([GitHub](https://github.com/OpenSourceMycetoma/Series-1-Fenarimols/issues))

I have, today, gone over to the Institute to make up the compound dilutions and put them through the assay. The assay was run in a 384 well plate, with each compound made at 20 uM, 6 uM, 2 uM and 0.6 uM concentrations. The samples were then diluted by 2 when put through the assay (i.e. 10 uM, 3 uM, 1 uM and 0.3 uM assay concentrations). Each concentration was then duplicated to 10 identical wells. DMSO controls were also used.

At first glance, there appears to be a mix of hERG binders and non-binders but full analysis of the results will occur early next week.

**Results are posted below.**
--------------------------------------------------
[hERG 2018 Chemdraw.zip](https://github.com/OpenSourceMalaria/Series4/files/1922647/hERG.2018.Chemdraw.zip)

**Comment by drc007** [comment]
It might be useful to look at a couple of compounds that were previously shown to be HERG active in this assay for comparison.

**Comment by MFernflower** [comment]
A bit unrelated but @mattodd How is the mycetoma project going? I have not seen any updates since it first launched?,

**Comment by edwintse** [comment]
@MFernflower Unfortunately, work on the mycetoma project here at USyd is on hold at the moment due to unforeseen circumstances (Matthew O'Dowd is no longer with the group). However, work will resume later in the year as we're expecting Hung to return to begin his PhD around July.

**Comment by mattodd** [comment]
OK, thanks @MFernflower . But as always with suggestions, please always _always_ include the "why" along with the suggestion (ideally with reference to lit or some external source). And if you wouldn't mind, post the suggestion and discussion over on that project, rather than here. Yes, we're pausing on new mycetoma chem, but it's just a pause. I believe the paper is almost accepted for publication, and we can use that as a means to identify new compounds that may already exist and which don't need more synthesis in the short term. Anything more on this --> MycetOS.

**Comment by mattodd** [comment]
Good suggestion @drc007 . OSM-S-369 is super-close to a previously tested compound, so we had thought that would be sufficient, but to cover our bases you're quite right, we should use an identical compound. On the list for a subsequent evaluation. It is a _serious_ piece of kit the Victor Chang guys have.

**Comment by MFernflower** [comment]
Is the data available for viewing yet? @edwintse

**Comment by edwintse** [comment]
@MFernflower Still waiting on it to come back but will make a post here when we do!

**Comment by edwintse** [comment]
**UPDATE:**
**Long story short, the assay needs to be run again (scheduled for the coming weeks) with more DMSO controls and also a known hERG blocker positive control (explanation below).**

Preliminary data was received, **HOWEVER**, there was an issue with the results. Four negative DMSO controls were used but only two of these gave useable information. The other two showed a significant run-down in the current. In addition to this, two of the compounds (**106** and **515**) appeared to show no concentration dependence to blocking suggesting this may also be due to run-down. The .doc from the Victor Chang guys is attached below. The hERG potencies (IC50) of the remaining compounds are shown below but they are not the final results.

![herg 2018 update](https://user-images.githubusercontent.com/18062981/39853180-ff6b839c-5463-11e8-8550-da8f97b81fa6.jpeg)
----------------------------------------------------------------------------------------------------
Below is the tabulated data extracted from the 2014 set of compounds. I've added this set for comparison. One thing that is not immediately apparent to me is how the 2014 set were deemed good/ok/bad. From what I've found/been told on the safety margins, a 30-fold change or greater between the hERG IC50 and the target IC50 is good (I've also been told these guidelines are currently under review and any updates have not been confirmed yet). Having calculated the fold change for the 2014 set, it looks like some that were identified as bad/ok hERG blockers are actually good? Is there something I'm missing? Do you have any thoughts on this @drc007?

![herg tabulated data](https://user-images.githubusercontent.com/18062981/39853343-1e1425b4-5465-11e8-9cdd-7d3d7eb46968.jpg)

[hERG tabulated data.xlsx](https://github.com/OpenSourceMalaria/Series4/files/1990464/hERG.tabulated.data.xlsx)
[Overview of Data_18-04-2018.docx](https://github.com/OpenSourceMalaria/Series4/files/1990435/Overview.of.Data_18-04-2018.docx)

**Comment by drc007** [comment]
@edwintse hERG is an issue that plagues almost every drug discovery program and the difficulty is trying to put everything into context. I've written a page on hERG on the Drug Discovery Resources website https://www.cambridgemedchemconsulting.com/resources/herg_activity.html

The bottom line is that in vitro assays are not a reliable way to predict safety, you must have in vivo data. For example you compare IC50 values in vitro, but in man I suspect for efficacy against the parasite you will want plasma levels of drug many times the IC50 (after all you don't want to only kill 50% of parasite). The 30-fold margin mentioned is based on in vivo data not in vitro. Until you know what the likely concentration will need to be in the clinic it is difficult to predict any safety window. 
Also remember QT prolongation can be seen with low levels of hERG channel occupancy, and will likely be driven by Cmax levels after oral dosing. In contrast you may find that you need to maintain many multiples of P. fal IC50 at trough for antimalarial activity.

**Comment by mattodd** [comment]
@edwintse could you please install a link to this page on the [wiki](https://github.com/OpenSourceMalaria/Series4/wiki/hERG-Activity) if it's not already there?

**Comment by edwintse** [comment]
Here is the summary of the results (_in vitro_ potency, hERG IC50 and hERG -pIC50 shown):

![herg 2018 summary](https://user-images.githubusercontent.com/18062981/42201975-45fcaf18-7edd-11e8-94e4-db9d2154acc5.png)

Data will be entered into the Master List and linked on the wiki.

Next plan of action is evaluating some of the [original compounds](https://github.com/OpenSourceMalaria/Series4/wiki/hERG-Activity) in this assay to validate the original results. We will also have a select number of new compounds evaluated as well.

**Comment by MFernflower** [comment]
Data has been added to the wiki @edwintse

**Comment by edwintse** [comment]
Data incorporated into wiki. Closing.
